libraries/AP_Logger/AP_Logger_File.cpp

```diff
@@ -583,11 +583,15 @@ uint32_t AP_Logger_File::_get_log_time(const uint16_t log_num)
             // it is the file we are currently writing
             free(fname);
             write_fd_semaphore.give();
+#if AP_RTC_ENABLED
```

Comment:
This would result in a non-zero-byte change to the code (or, in the case of our "normal vehicle" firmwares  the no-compiler-output-change status of this PR would be lost)